package jumpingstatements;

public class ContinueExample {
    public static void main(String[] args){
        for (int i=1; i<=5; i++){
            if(i==2){
                continue;
            }
            System.out.println("The value is:" +i);
        }
    }
}
